app/src/main/java/cm/aptoide/pt/smart/appfiltering/data/Age.java

```diff
@@ -0,0 +1,52 @@
+
+package cm.aptoide.pt.smart.appfiltering.data;
```

Comment:
cm.aptoide.pt.smart.appfiltering.data This package only contains POJO objects for json response.